### PR TITLE
MySQL: expose storage_type, iops and deprecate cores

### PR DESCRIPTION
**Pull request by zucchinidev**
[#183023678](https://www.pivotaltracker.com/story/show/183023678)

Which other branches should this be merged with (if any)?

None

### CODE REVIEW

File: reference/aws-mysql.html.md.erb [hunk 1]
```diff
@@ -37,18 +37,18 @@ The following table lists parameters which can only be configured for additional
...
-| `plan_updateable`    | Whether the Plan supports upgrading, downgrading, or sidegrading to another version.  | true     |
-| `cores`*             | Minimum number of cores for service instance. 1&ndash;64, multiples of 2.             | 2     |
+| `free`               | When false, service instances of this service plan have a cost.                       | true      |
+| `bindable`           | Specifies whether Service Instances of the Service Plan can bind to applications.     | true      |
+| `plan_updateable`    | Whether the Plan supports upgrading, downgrading, or sidegrading to another version.  | true      |
+| `cores`              | Deprecated - Minimum number of cores for service instance. 1&ndash;64, multiples of 2. Use `instance_class` instead. | 2         |
 | `mysql_version`*     | The version for the MySQL instance.                                                   | _n/a_     |
-| `storage_gb`*        | Size of storage volume for service instance. 5&ndash;4096.                            | 5     |
+| `storage_gb`*        | Size of storage volume for service instance. 5&ndash;4096.                            | 5         |
```
Comment (FelisiaM) [inline reference/aws-mysql.html.md.erb:45]: Worth changing the new default to 100 as well just to be consistent

File: release-notes.html.md.erb [hunk 2]
```diff
@@ -15,7 +15,12 @@ These are release notes for <%= vars.product_full %>.
 
 This release has the following breaking changes:
 
-- **Run-in heading:** Description
+- **the default storage type is now set as 'io1' (provisioned IOPS SSD):** Previously the default used
```
Comment (FelisiaM) [inline release-notes.html.md.erb:18]: Need to add the service it relates to in the title. E.g `RDS MySQL new default storage type is provisioned IOPS SSD : ....`

File: release-notes.html.md.erb [hunk 3]
```diff
@@ -45,6 +50,18 @@ New features and changes in this release:
 - **`deletion_protection` is configurable and updatable:** New added property that allows you to configure the
   deletion protection for the MySQL database in both the provisioning and the updating phase of the instance.
   By default the protection is disabled.
+
+- **`storage_type` and `iops` are configurable and updatable:** New added property that allows you to configure the
```
Comment (FelisiaM) [inline release-notes.html.md.erb:54]: Maybe worth mentioning that default storage type is io1 if none is defined

File: release-notes.html.md.erb [hunk 4]
```diff
@@ -15,7 +15,12 @@ These are release notes for <%= vars.product_full %>.
 
 This release has the following breaking changes:
 
-- **Run-in heading:** Description
+- **the default storage type is now set as 'io1' (provisioned IOPS SSD):** Previously the default used
+  'gp2' (general purpose SSD).
+  Users who previously had custom plans should add the property `"storage_type":"gp2"` to the plan definition,
+  to ensure the storage type is not amended on any update.
+  As part of this work, the default storage size has also been increased to 100GB, as this is the smallest
+  storage supported by the 'io1' storage type.
```
Comment (FelisiaM) [inline release-notes.html.md.erb:23]: Can add a link to the "changing custom plans" page here

File: release-notes.html.md.erb [hunk 5]
```diff
@@ -15,7 +15,12 @@ These are release notes for <%= vars.product_full %>.
...
 
 ### Features
```
Comment (FelisiaM) [inline release-notes.html.md.erb:25]: A meta point for all the release notes - its worth putting all the MySQL related notes under the same bullet point with an increased indentation (so its clear these properties relate to Myssql and also Aurora release notes can also be distinctive) 

See example in previous release notes  https://docs.vmware.com/en/Tanzu-Cloud-Service-Broker-for-AWS/1.2/csb-aws/GUID-release-notes.html

**Review by FelisiaM — CHANGES_REQUESTED** [review]

File: release-notes.html.md.erb (hunk 5 above)
Comment (zucchinidev) [reply release-notes.html.md.erb:28]: All fixed!! Thank you very much for your help!

**Review by FelisiaM — APPROVED** [review]

**Comment by RichardJJG** [comment]
Thanks, @zucchinidev and @FelisiaM

**Pull request merged**